src/pages/App/App.module.scss

```diff
@@ -1,17 +1,21 @@
 .visualmusic {
     display: flex;
     flex-direction: column;
-    padding-bottom: 64px;
+    // padding-bottom: 64px;
```

Comment:
This is line will be removed on [PR #39](https://github.com/zero-to-mastery/visual-music/pull/39). So you can either remove it or leave it.